packages/manager/apps/web/client/app/hosting/multisite/hosting-multisite.service.js

```diff
@@ -59,7 +59,10 @@ angular.module('services').service(
               });
             },
           );
+        } else {
+          throw new Error(response.messages[0]?.message);
         }
```

Comment:
IMO, it should be better to do the fix on the 2API side and return error 500 when an error occurs